lib/compressTextureCoordinates.js

```diff
@@ -0,0 +1,172 @@
+'use strict';
+var Cesium = require('cesium');
+var AttributeCompression = Cesium.AttributeCompression;
+var Cartesian2 = Cesium.Cartesian2;
+var DeveloperError = Cesium.DeveloperError;
+var ShaderSource = Cesium.ShaderSource;
+var WebGLConstants = Cesium.WebGLConstants;
+var defined = Cesium.defined;
+
+var byteLengthForComponentType = require('./byteLengthForComponentType');
+var findAccessorMinMax = require('./findAccessorMinMax');
+var getAccessorByteStride = require('./getAccessorByteStride');
+var numberOfComponentsForType = require('./numberOfComponentsForType');
+var readBufferComponentType = require('./readBufferComponentType');
+var writeBufferComponentType = require('./writeBufferComponentType');
+var uninterleaveAndPackBuffers = require('./uninterleaveAndPackBuffers');
+
+module.exports = compressTextureCoordinates;
+
+function compressTextureCoordinates(gltf) {
```

Comment:
Include some doc here like this: https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Core/AttributeCompression.js#L233